EU4toV2/Source/V2World/V2World.h

```diff
@@ -101,7 +101,9 @@ class World
 	void convertTechs(const EU4::World& sourceWorld);
 	void allocateFactories(const EU4::World& sourceWorld);
 	void setupPops(const EU4::World& sourceWorld);
-	void addUnions();
+	void addUnions(bool hreDecentralized, const std::string& emperorCulture);
+	void decentralizeHRE(bool hreDecentralized, const std::string& emperorCulture);
+	std::shared_ptr<Country> findEmperorHRE();
```

Comment:
[[nodiscard]] std::shared_ptr<Country> findEmperorHRE() const;
You're not changing anything.